lib/storage/storage-s3.js

```diff
@@ -47,6 +47,20 @@ class StorageS3 extends StorageBase {
         this.dynamoDb = new AWS.DynamoDB();
     }
 
+    getSources(content) {
```

Comment:
Not important now, but we should probably extract this kind of stuff to a "GL utils" sometime. I bet we'll need more of these...